style.css

```diff
@@ -0,0 +1,783 @@
+  flex-direction: row;
+  list-style: none;
+  margin-right: 3em;
+}
+
+.nav6 {
+  border: #ec5242 5px solid;
+  padding: 5px;
+  border-radius: 5px;
+  font-weight: bold;
+  margin-left: 2em;
+}
+
+.items-port:hover {
+  color: #fff;
+}
+
+.items-about:hover {
+  color: #fff;
+}
+
+.items-contact:hover {
+  color: #fff;
+}
+
+.items-news:hover {
+  color: #fff;
+}
+
+.nav1:hover {
+  color: #ec5242;
+}
+
+.nav2:hover {
+  color: #ec5242;
+}
+
+.nav3:hover {
+  color: #ec5242;
+}
+
+.nav4:hover {
+  color: #ec5242;
+}
+
+.nav5:hover {
+  color: #ec5242;
+}
+
+.nav6:hover {
+  color: #ec5242;
+  box-shadow: 0 0 15px 3px #ec5242;
+}
+
+.speaker {
+  background: #fff;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  max-width: 800px;
+  width: 100%;
+  height: 100%;
+  padding: 0 2em;
+}
+
+.teacher-topic {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1em;
+}
+
+.teacher-list {
+  margin: 40px auto;
+  width: 350px;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.teacher-img {
+  width: 80px;
+  height: 80px;
+  margin-right: 15px;
+  box-shadow: -10px -10px 5px -3px rgba(0, 0, 0, 0.28);
+}
+
+.teacher-header {
+  color: #1d1d1d;
+  font-size: 1rem;
+  font-weight: 200;
+}
+
+.teacher-profession {
+  color: #ec5242;
+  font-size: 11px;
+  margin: 5px 0 5px 0;
+}
+
+.teacher-line {
+  width: 25px;
+  margin: 5px 0;
+}
+
+.teacher-description {
+  font-size: 11px;
+  font-weight: 100;
+}
+
+.about-section-main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: 100%;
+  height: 100%;
+  padding: 0 2em;
+}
+
+.about-email {
+  margin-top: 1em;
+}
+
+.para1-about {
+  font-weight: 700;
+  margin-bottom: 1em;
+  width: 50%;
+  margin-top: -1em;
+}
+
+.about-orange {
+  color: #ec5242;
+  font-size: 18px;
+}
+
+.about-about-page {
+  margin-top: 1em;
+}
+
+.about-lower {
+  margin-top: 3em;
+}
+
+.about-discuss {
+  margin-top: 3em;
+  width: 50%;
+}
+
+.about-hr {
+  width: 100%;
+  height: 6px;
+  background-color: #ec5242;
+  border: none;
+}
+
+.ice-img {
+  width: 300px;
+  height: 300px;
+}
+
+.about-botttom-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  margin-top: 4em;
+}
+
+.line {
+  width: 5rem;
+  height: 3px;
+  background-color: #ec5242;
+  margin: 1.5rem auto;
+}
+
+.line2 {
+  width: 5rem;
+  height: 3px;
+  background-color: #ec5242;
+}
+
+.aboutchef-img {
+  height: 300px;
+  filter: invert(17%) sepia(96%) saturate(1393%) hue-rotate(10deg) brightness(97%) contrast(79%);
+  margin-left: 3em;
+  vertical-align: middle;
+}
+
+.aboutchef-img2 {
+  height: 300px;
+  filter: invert(17%) sepia(96%) saturate(1393%) hue-rotate(10deg) brightness(97%) contrast(79%);
+  margin-left: 3em;
+}
+
```

Comment:
- [ ] Kindly center the image on the screen in the mobile version by adding `max-width: 350px;` for both images and removing the margin.
> Your design
>![image](https://user-images.githubusercontent.com/53402600/191809810-27b0fd34-26fe-4416-836b-0c45fa17b39c.png)

Reply:
ok I fixed it